frontend/src/commons/Favorite.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { CgRemoveR } from 'react-icons/cg';
+
+export default function Favorite({ favorite, className }) {
+  return (
+    <FavoriteStyled className={className}>
+      <FavoriteHeader>
+        <LogoStyled src={favorite.crestUrl} alt="Team Logo" />
+        {favorite.name}
+        <RemoveButtonStyled>{<CgRemoveR />}</RemoveButtonStyled>
+      </FavoriteHeader>
+      <MatchStyled>
+        <div class="date">Sa 28.11.2020 15:30 Uhr</div>
+        <div>{favorite.lastMatch.homeTeam}</div>
+        <div class="result">
+          {favorite.lastMatch.homeTeamGoals} :{' '}
+          {favorite.lastMatch.awayTeamGoals}
+        </div>
+        <div>{favorite.lastMatch.awayTeam}</div>
+      </MatchStyled>
+      <MatchStyled>
+        <div className="date">Mi 02.12.2020 21:00 Uhr</div>
+        <div>{favorite.currentMatch.homeTeam}</div>
+        <div class="result">
+          {favorite.currentMatch.homeTeamGoals} :{' '}
+          {favorite.currentMatch.awayTeamGoals}
+        </div>
+        <div>{favorite.currentMatch.awayTeam}</div>
+      </MatchStyled>
+      <MatchStyled>
+        <div className="date">Sa 05.12.2020 15:30 Uhr</div>
+        <div>{favorite.nextMatch.homeTeam}</div>
+        <div class="result">
+          {favorite.nextMatch.homeTeamGoals} :{' '}
+          {favorite.nextMatch.awayTeamGoals}
+        </div>
+        <div>{favorite.nextMatch.awayTeam}</div>
+      </MatchStyled>
+    </FavoriteStyled>
+  );
+}
+
+const FavoriteStyled = styled.section`
+  padding: var(--size-s);
+  box-shadow: 2px 2px 2px var(--blue-75);
+  border: var(--blue-border);
+  border-radius: var(--size-s);
+  background: #eff8fb;
+`;
+
+const FavoriteHeader = styled.header`
+  display: grid;
+  grid-template-columns: 1fr 6fr 1fr;
+  align-items: center;
+  padding-bottom: var(--size-s);
+  color: var(--blue-main);
+  text-align: center;
+  font-weight: bold;
+  font-size: 18px;
```

Comment:
Better set `font-size` in `em` or `rem` keeps it more consistent and doesn't mess with user expectations